packages/chaire-lib-backend/src/models/db/users.db.queries.ts

```diff
@@ -68,7 +77,7 @@ const find = async (
         }
         const query = knex(tableName);
         if (whereData.usernameOrEmail !== undefined) {
-            query.where('email', whereData.usernameOrEmail);
+            query.whereILike('email', whereData.usernameOrEmail);
```

Comment:
You probably have to add the whereILike in the below for the email and username cases. The forEach probably needs to be changed to something specific for those 2 fields, like it is done here.